app/controllers/api/v1/forecast_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::ForecastController < ApplicationController
+  def index
+  location = MapQuestService.new.get_coordinates(params[:location])
+  lat = location[:results][0][:locations][0][:latLng][:lat]
+  lon = location[:results][0][:locations][0][:latLng][:lng]
```

Comment:
I tried to make a Location Poro with this exact information except it didn't work. However, when I put it in the index, it ended up working and pulling the coordinates. Possible refactor over here.